inkfish/cmds.py

```diff
@@ -15,10 +19,12 @@ def create_pot_parser():
         description='Generate or verify a proof of time using the Chia ' +
                     'Verfiable Delay Function (VDF)',
     )
-    parser.add_argument("-s", "--small", action="store_true",
-                        help="Indicate creation of a small proof.")
+    parser.add_argument("-t", "--type", default="wesolowski",
+                        help="the type of proof, wesolowski, n-wesolowski, or pietrzak")
```

Comment:
Use "choices" here and then you don't need to check for invalid proof type. https://docs.python.org/3/library/argparse.html#choices